docs/data-fetching.md

```diff
@@ -118,7 +118,7 @@ function defaultFieldResolver($source, $args, $context, \GraphQL\Type\Definition
         }
     }
 
-    return $property instanceof \Closure ? $property($source, $args, $context) : $property;
+    return is_callable($property) ? $property($source, $args, $context, $info) : $property;
```

Comment:
We actually reverted `is_callable()` here back to `instanceof Closure`. Please revert this part and I'll merge. Thanks for noticing!